relay-kafka/src/config.rs

```diff
@@ -155,7 +149,6 @@ impl Default for TopicAssignments {
             transactions: "ingest-transactions".to_owned().into(),
             outcomes: "outcomes".to_owned().into(),
             outcomes_billing: None,
-            sessions: "ingest-sessions".to_owned().into(),
```

Comment:
Does the ops repo still have configuration for this? We don't want Relay to crash because of an unknown topic parameter.

Reply:
Yes. https://github.com/getsentry/ops/blob/master/k8s/clusters/us/default.yaml#L3256-L3257. Though is this comment actually backwards and it needs to be removed from ops first?